distributions/openhab/src/main/resources/bin/restore.ps1

```diff
@@ -0,0 +1,211 @@
+﻿#Requires -Version 5.0
+Set-StrictMode -Version Latest
+
+Function Restore-openHAB {
+    <#
+    .SYNOPSIS
+    Restores openHAB files from a backup file.
+    .DESCRIPTION
+    The Restore-openHAB function performs the necessary tasks to restore openHAB from a backup file.
+    .PARAMETER OHDirectory
+    The directory where openHAB is installed (default: current directory).
+    .PARAMETER OHBackups
+    The directory where backup the files are.
+    .PARAMETER FileName
+    The name of the backup file to use (do not specify to use the latest)
+    .PARAMETER AutoConfirm
+    Whether to auto confirm ($true) replacement of files (used for headless mode)
+    .EXAMPLE
+    Restore an openHAB instance from the latest backup file
+    Restore-openHAB
+    .EXAMPLE
+    Restore the openHAB distribution in the C:\openHAB2 directory from c:\openHAB2-backup\backup.zip without any user interaction
+    Restore-openHAB -OHDirectory C:\openHAB2 -OHBackups c:\openHAB2-backup -FileName backup.zip -AutoConfirm $true
+    #>
+
+    [CmdletBinding()]
+    param(
+        [Parameter(ValueFromPipeline = $True)]
+        [string]$OHDirectory = ".",
+        [Parameter(ValueFromPipeline = $True)]
+        [string]$OHBackups = "",
+        [Parameter(ValueFromPipeline = $True)]
+        [string]$FileName = "",
+        [Parameter(ValueFromPipeline = $True)]
+        [boolean]$AutoConfirm = $False
+    )
+
+    begin {}
+    process {
+
+        Import-Module $PSScriptRoot\common.psm1 -Force
+
+        Write-Host ""
+        BoxMessage "openHAB 2.x.x restore script" Magenta
+        Write-Host ""
+        
+        try {
+            $StartDir = Get-Location -ErrorAction Stop 
+        }
+        catch {
+            return PrintAndReturn "Can't retrieve the current location - exiting" $_
+        }
+
+        CheckForAdmin
+        CheckOpenHABRunning
+
+        Write-Host -ForegroundColor Cyan "Checking the specified openHAB directory"
+        $OHDirectory = GetOpenHABRoot $OHDirectory
+        if ($OHDirectory -eq "") {
+            return PrintAndReturn "Could not find the userdata directory! Make sure you are in the openHAB directory or specify the -OHDirectory parameter!"
+        }
+    
+        $OHConf = "$OHDirectory\conf"
+        $OHUserdata = "$OHDirectory\userdata"
+
+        if (([string]::IsNullOrEmpty($OHBackups))) {
+            $OHBackups = "$OHDirectory\backups"
+        }
+
+        if ([string]::IsNullOrEmpty($FileName)) {
```

Comment:
It would be nice to have something here like:

`Write-Host -ForegroundColor Yellow "No backup file specified. Finding latest backup."`